deps/rabbitmq_auth_backend_oauth2/src/rabbit_oauth2_schema.erl

```diff
@@ -25,6 +25,15 @@
     translate_scope_aliases/1
 ]).
 
+-define(RESOURCE_SERVERS_SYNONYMS, #{
+  "additional_scopes_key" => "extra_scopes_source"
+}).
+
+resource_servers_key_synonym(Name) ->
+  case maps:find(Name, ?RESOURCE_SERVERS_SYNONYMS) of {ok, Synonym} -> Synonym;
```

Comment:
This is a highly unusual formatting. Please add a line break after `of` and indent the `{ok, Value}` match head accordingly.

You probably want `maps:get/3` with a default anyway.